CHANGELOG

```diff
@@ -108,3 +108,11 @@
   - Fixes issue with encryption, byte-compilation (Thank you, Christian Schwarzgruber!)
   - Improves documentation (Thank you, EFLS!)
   - Fixes initialization error (Thank you, Christian Schwarzgruber!)
+
+## 2019-10-30 V2.0.0
```

Comment:
@cslux Feel free to add your name to the headline! You deserve it!

You could also add a line to the changelog that acknowledges the addition of maintainers, if you want.